hail/src/main/scala/is/hail/io/vcf/HtsjdkRecordReader.scala

```diff
@@ -18,10 +19,19 @@ class BufferedLineIterator(bit: BufferedIterator[String]) extends htsjdk.tribble
   }
 }
 
-class HtsjdkRecordReader(val callFields: Set[String]) extends Serializable {
+class HtsjdkRecordReader(val callFields: Set[String], val entryFloatTypeName: Option[String]) extends Serializable {
 
   import HtsjdkRecordReader._
 
+  val entryFloatType: TNumeric = entryFloatTypeName match {
```

Comment:
will become simpler if this isn't an option (see above).

It's also checked in Python, so the base case here is unnecessary but can definitely stay.

Reply:
Fixed.